backend/mailer/mailer.js

```diff
@@ -87,10 +88,15 @@ function sendQueueFailedEmail(err) {
 	}
 }
 
-function sendVerifyUserEmail(to, data) {
+async function sendVerifyUserEmail(to, data) {
+
+	const User = require("../models/user");
 
 	data.url = getURL("verify", {token: data.token, username: data.username, pay: data.pay});
 
+	const user = await User.findByUserName(data.username);
```

Comment:
Instead of doing a query here, get the function that calls sendVerifyUserEmail to give you the user's first name within the data object.

The function already has all these information. There's no need for another db query.